packages/connect-popup/src/static/popup.html

```diff
@@ -3254,9 +3251,22 @@ <h3>Enter your passphrase on <span class="device-name"></span></h3>
             };
             window.addEventListener('message', handlePopupContentScriptLoaded);
 
+            // The handshake is sent after the bootstrap message, so we process it here to avoid delays loading popup.js
+            // In the future, channel handling should be done in popup.js directly.
```

Comment:
so this is here because of backwards compatibility right? lets come up with some unified comment format that will help us remove these later. any ideas?

```
// todo: connect10
```
or something like this? 

this task is on me. I should define how to start collecting all these places systematically